portal/templates/CourseCreation.html

```diff
@@ -0,0 +1,25 @@
++{% extends "layouts/base.html" %}
++
++{% block title %}Course Edit{% endblock %}
++
++{% block content %}
++<form method='post'>
++  <label for="name">
++    <input type = "text" name = "course-name" id='name' />
++    </label>
```

Comment:
Another indentation issue. Once these are fixed, might need to double check the rest of the indentation